mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/layout/Size.kt

```diff
@@ -719,6 +758,27 @@ private class WrapContentModifier(
 		}
 	}
 
+	override fun equals(other: Any?): Boolean {
+		if (this === other) return true
+		if (other === null) return false
+		if (this::class != other::class) return false
+
+		other as WrapContentModifier
```

Comment:
Auto-generated via Android Studio(
Since the others will disappear from the file, I think we can leave this one as it is, while it seems that the implementation is effective.
But the fact is that it is implemented differently in different places of the project and I would like to have one common approach